Dockerfile

```diff
@@ -134,7 +136,8 @@ RUN set -ex && apt-get update && apt-get install -y --no-install-recommends \
     libssl1.1 \
     openjdk-11-jre \
     dos2unix \
-    gawk
+    gawk \
+    python3-pip
```

Comment:
I _think_ that you can install this in line 45 and then use it to install `mypy-protobuf`, so that pip does not need to be part of the final image, though I suppose `mypy-protobuf` will then need to be copied over to the new target from wherever it's installed.

Reply:
maybe that can work but I have doubts. what we are after is the `protoc-gen-mypy` plugin. when I install that on my system I see that it is a script referring things in my local python environment:
```
$ which protoc-gen-mypy
/home/maxjakob/.pyenv/shims/protoc-gen-mypy
$ cat /home/maxjakob/.pyenv/shims/protoc-gen-mypy
#!/usr/bin/env bash
set -e
[ -n "$PYENV_DEBUG" ] && set -x

program="${0##*/}"

export PYENV_ROOT="/home/maxjakob/.pyenv"
exec "/home/maxjakob/.pyenv/libexec/pyenv" exec "$program" "$@"
```
I'm going to give it a shot and see if I can find all related things to copy from the `build` to the `protoc-all` stage to keep the container size slimg.

Reply:
I think I made it work.